transformation/src/jade-schema/tables/pipeline_activity.table.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "pipeline_activity",
+  "columns": [
```

Comment:
Should this include a pointer back to the experiment?

Reply:
Kathy says no